frontend/src/lib-common/utils/useAxiosResponseInterceptor.ts

```diff
@@ -0,0 +1,32 @@
+// SPDX-FileCopyrightText: 2017-2021 City of Espoo
+//
+// SPDX-License-Identifier: LGPL-2.1-or-later
+
+import axios, { AxiosInstance } from 'axios'
+import { useEffect, useState } from 'react'
+
+export function useAxiosResponseInterceptor(client: AxiosInstance): {
+  lastResponse: number
+} {
```

Comment:
I don't know if it makes sense to have this as an exported custom hook as it is. I would either move it to `usePeriodicRefresh` and not export it or change it so that it receives a callback as a parameter and returns its result so that it could be reused in cases where you might need a response interceptor to do something.